stories/link/link.stories.js

```diff
@@ -3,38 +3,76 @@ import '../../dist/link.css';
 export default {
     title: 'Components/Link',
     parameters: {
-        description: 'Link component supports the use of modifiers, as well as the <code>is-visited</code> flag, which allows for optimal customization of your hyperlink. There are currently no restrictions as to which content can be included in this component.',
+        description: `A link (also known as hyperlink) is an interactive text element that allows users to navigate to another page or trigger a certain event.
+
+Note: Link supports the <code>is-visited</code> flag, which allows for optimal customization.
```

Comment:
This note doesn't say anything.
`is-visited` class can be used to manually style links as "visited" if HTML element or the browser doesn't support `:visited` state.